### ISSUE TITLE
Oligodendroglioma

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6735]
**Preferred term label:**
Oligodendroglioma

**Synonyms**

**Definition (free text, please give PubMed ID)**
A form of glioma accounting for less than 10% of diffuse gliomas, and often presenting with focal-onset seizures. They are typically WHO grade I or III, with characteristic histological appearances and complete chromosome 1p and 19q codeletion.

25943885

**Parent term (use hpo.jax.org/app)**
Glioma

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
added
